contents/handbook/people/team-structure/core-experience.md

```diff
@@ -66,6 +66,9 @@ List below is prioritized.
 Our current focus is making Product Managers at scale-ups successful.
 We'll still strive to **maintain** success with technical users.
 
+## Output metrics
+
+- Primary: [Discoveries](https://app.posthog.com/insights/O4R-Frza?events=%5B%5D&actions=%5B%7B%22id%22%3A%2210784%22%2C%22math%22%3A%22total%22%2C%22name%22%3A%22Discoveries%22%2C%22type%22%3A%22actions%22%2C%22order%22%3A0%7D%5D&display=ActionsLineGraph&insight=TRENDS&interval=week&date_from=-90d&new_entity=%5B%5D&properties=%5B%5D&breakdown_type&filter_test_accounts=true#fromDashboard=20464)
```

Comment:
open q: Should one of our core teams have their primary metric be based around engagement / retention instead of both focused on discoveries?

Reply:
That's interesting @kpthatsme, I do wonder what are you thinking in terms of engagement. I think we could potentially be measuring engagement metrics as inputs to improve, and retention as output. My concern with retention is that there are so many flavors and nuances that I'm not sure how feasible would it be to actually have one team focused on this.

Reply:
I thought we'd consider discoveries as ah-ha moments, thus rational for engagement.

Open to specific suggestions if we need to split out engagement from discoveries as a new PR.